modules/application/src/main/java/edu/northeastern/truthtree/assembler/StoriesAssembler.java

```diff
@@ -0,0 +1,46 @@
+package edu.northeastern.truthtree.assembler;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.io.IOException;
+import java.util.List;
+
+import edu.northeastern.truthtree.dto.StoryDTO;
+import edu.northeastern.truthtree.dto.StoryDTOWrapper;
+import edu.northeastern.truthtree.dto.StoryDTOWrapperList;
+
+
+@Component
+public class StoriesAssembler {
+
+  private ObjectMapper mapper;
+
+  @Autowired
+  public StoriesAssembler(ObjectMapper mapper) {
+    this.mapper = mapper;
+
+  }
+
+  public List<StoryDTO> fromJSONStringToDTOList(String jsonString) {
+    StoryDTOWrapperList storyDTO = new StoryDTOWrapperList();
+    try {
+      storyDTO = mapper.readValue(jsonString, StoryDTOWrapperList.class);
+    } catch (IOException e) {
+      e.printStackTrace();
+    }
+    return storyDTO.getData();
+  }
+
+  public StoryDTO fromJSONStringToDTO(String jsonString) {
+    StoryDTOWrapper storyDTO = new StoryDTOWrapper();
+    try {
+      storyDTO = mapper.readValue(jsonString, StoryDTOWrapper.class);
```

Comment:
```
Map<String, Object> bodyMap = mapper.readValue(jsonString, new TypeReference<HashMap<String, Object>>(){});
StoryDTO realStoryDTO = new StoryDTO();
realStoryDTO.setContent((String) bodyMap.get("content"));
...
etc.
```
Doing this eliminate the need for `StoryDTOWrapper` and `StoriesAssembler`

Reply:
This may work because there is just one response dto in the content.